build-release-tools/application/make_debian_packages.py

```diff
@@ -278,12 +335,18 @@ def main():
     Exit on encountering any error.
     """
     args = parse_args(sys.argv[1:])
+    # Bintray is the offical/final place to store the deb
     bintray = Bintray(args.bintray_credential, args.bintray_subject, args.bintray_repo)
+    # Artifactory is the staging/temp place to store the deb between deb-build and post-test steps
+    artifactory = JFrogArtifactory( user_cred=(args.artifactory_username, args.artifactory_password), artifactory_loc= args.artifactory_url)
 
     checkout_repos(args.manifest_file, args.build_directory, args.force, args.jobs, git_credential=args.git_credential)
     all_repos = get_build_repos(args.build_directory)
-    package_not_exist = create_packages_filter(bintray, args.build_directory, args.is_official_release)
+    package_not_exist = create_packages_filter(bintray, artifactory, args.artifactory_repo, args.build_directory, args.is_official_release)
     package_need_build_repos = filter(package_not_exist, all_repos)
+    package_need_build_repos.append('RackHD') #always rebuild rackhd.deb
```

Comment:
so careful review.@changev !
I will remove it.
I checked from the console log
RackHD did show up twice:
```
15:51:36 [Info] on-taskgraph version 2.5.0-20170513UTC-b573ca4 no cache version available on Bintray. Do deb build.
15:51:36 [Info] Repo on-core will run deb-build 
15:51:36 [Info] Repo on-http will run deb-build 
15:51:36 [Info] Repo on-statsd will run deb-build 
15:51:36 [Info] Repo on-tftp will run deb-build 
15:51:36 [Info] Repo di.js will run deb-build 
15:51:36 [Info] Repo on-wss will run deb-build 
15:51:36 [Info] Repo on-skupack will run deb-build 
15:51:36 [Info] Repo on-tasks will run deb-build 
15:51:36 [Info] Repo ucs-service will run deb-build 
15:51:36 [Info] Repo RackHD will run deb-build 
15:51:36 [Info] Repo on-syslog will run deb-build 
15:51:36 [Info] Repo on-dhcp-proxy will run deb-build 
15:51:36 [Info] Repo on-taskgraph will run deb-build 
15:51:36 [Info] Repo RackHD will run deb-build 
```